src/mono/mono.proj

```diff
@@ -263,6 +263,7 @@
       <_MonoCMakeArgs Include="-DCMAKE_INSTALL_PREFIX=&quot;$(MonoObjDir.Replace('\','/'))out&quot;"/>
       <_MonoCMakeArgs Include="-DCMAKE_INSTALL_LIBDIR=lib"/>
       <_MonoCMakeArgs Include="-DCMAKE_BUILD_TYPE=$(Configuration)"/>
+      <_MonoCMakeArgs Include="-DCLR_CMAKE_KEEP_NATIVE_SYMBOLS=true" Condition="'$(KeepNativeSymbols)' == 'true' or '$(BuildMonoAOTCrossCompilerOnly)' == 'true'" />
```

Comment:
no idea why we didn't strip the AOT compilers before, was that intentional? I can't think of a reason why.

Reply:
Lets enable it to find out what CI thinks. 😅